docs/concepts/glossary.md

```diff
@@ -0,0 +1,166 @@
+per running [Flink Job](./glossary#flink-job).
+
+#### Operator
+
+Node of a [Logical Graph](./glossary#logical-graph). An Operator performs a certain operation,
+which is usually executed by a [Function](./glossary#function). Sources and Sinks are special
+Operators for data ingestion and data egress.
+
+#### Operator Chain
+
+An Operator Chain consists of one or more consecutive [Operators](./glossary#operator) without any
+repartitioning in between. Operators within the same Operation Chain forward records to each other
+directly without going through serialization or Flink's network stack.
+
+#### Partition
+
+A partition is an independent subset of the overall data stream or data set. A data stream or
+data set is divided into partitions by assigning each [record](./glossary#Record) to one or more
+partitions. Partitions of data streams or data sets are consumed by [Tasks](./glossary#task) during
+runtime. A transformation which changes the way a data stream or data set is partitioned is often
+called repartitioning.
+
+#### Physical Graph
+
+A physical graph is the result of translating a [Logical Graph](./glossary#logical-graph) for
+execution in a distributed runtime. The nodes are [Tasks](./glossary#task) and the edges indicate
+input/output-relationships or [partitions](./glossary#partition) of data streams or data sets.
+
+#### Record
+
+Records are the constituent elements of a data set or data stream.
+[Operators](./glossary#operator) and [Functions](./glossary#Function) receive records as input
+and emit records as output.
+
+#### Flink Session Cluster
+
+A long-running [Flink Cluster](./glossary#flink-cluster) which accepts multiple
+[Flink Jobs](./glossary#flink-job) for execution. The lifetime of this Flink Cluster is not bound
+to the lifetime of any Flink Job. Formerly, a Flink Session Cluster was also known as a Flink Cluster in
+*session mode*. Compare to [Flink Application Cluster](./glossary#flink-application-cluster).
+
+#### State Backend
+
+For stream processing programs, the State Backend determines how state is stored on each Taskmanager
+(Java Heap of Taskmanager or (embedded) RocksDB) as well as where it is written upon a checkpoint
+(Java Heap of Flink Master or Filesystem).
+
+#### Sub-Task
+
+A Sub-Task is an instance of [Task](./glossary#task) responsible for processing a
+[partition](./glossary#partition) of the data stream. The term "Sub-Task" emphasizes that there are
+multiple parallel Tasks for the same [Operator(s)](./glossary#operator).
+
+#### Task
+
+Node of a [Physical Graph](./glossary#physical-graph). A task is the basic unit of work, which is
+executed by Flink's runtime. Tasks encapsulate exactly one parallel
+[Operator Chain](./glossary#operator-chain).
+
+#### Flink TaskManager
+
+TaskManagers are the worker processes of a [Flink Cluster](./glossary#flink-cluster).
+[Tasks](./glossary#task) are scheduled to TaskManagers for execution.
+
+#### Transformation
```

Comment:
Mention that a Transformation is an API concept?

Reply:
Can you elaborate?

Reply:
From the description I read that the term "Transformation" is used in the context of the API, i.e., when implementing an application. When the application is executed it is rather operator or task. If this distinction is not intended, how would a transformation relate to operator/task/function?

Reply:
Okay, I see. I have added a sentence. Let me know, if this fits.